### PR TITLE
Update release process

### PR DESCRIPTION
**Pull request by eheimbuch**
## Proposed changes

Updates the SCM-Manager release process after the transition to gradle as build tool.

### Your checklist for this pull request

- [x] Documentation updated (only necessary for new features or changed behaviour)

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: docs/en/release-process.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ git commit -m "Adjust changelog for release <version>"
 
 Jenkins will
 
-- update `pom.xml` and `package.json`
+- update `build.gradle` and `package.json`
```
Comment (sdorra) [inline docs/en/release-process.md:45]:
```suggestion
- update `gradle.properties` and `package.json`
```

File: docs/en/release-process.md [hunk 2]
```diff
@@ -74,7 +78,7 @@ git merge origin/support/<support branch>
 
 If you need to update the parent of the plugin to a new release of SCM-Manager, change it now:
 
-- `pom.xml`: `parent.version`
+- `build.gradle`: `parent.version`
```
Comment (sdorra) [inline docs/en/release-process.md:81]:
```suggestion
- `build.gradle`: `scmVersion`
```

File: docs/en/release-process.md [hunk 3]
```diff
@@ -84,8 +88,8 @@ Check if all plugin dependencies are proper versions and not SNAPSHOT!
 ## Build, commit and push
 
 ```
-rm -rf node_modules yarn.lock && mvn clean install \
-&& git add yarn.lock pom.xml package.json \
+rm -rf node_modules && ./gradlew build \
```
Comment (sdorra) [inline docs/en/release-process.md:91]: I think we don't need the removal of `node_modules` any more

File: docs/en/release-process.md [hunk 4]
```diff
@@ -109,13 +113,14 @@ export VERSION=<version> \
 
 Jenkins will
 
-- update versions in pom.xml and package.json
+- update versions in build.gradle and package.json
```
Comment (sdorra) [inline docs/en/release-process.md:116]:
```suggestion
- update versions in gradle.properties and package.json
```

**Review by sdorra — CHANGES_REQUESTED** [review]

File: docs/en/release-process.md (hunk 3 above)
Comment (eheimbuch) [reply docs/en/release-process.md:91]:
```suggestion
./gradlew build \
```

**Pull request merged**